tests/integration/run.bats

```diff
@@ -219,3 +219,21 @@ EOF
 	[ ${#lines[@]} -eq 1 ]
 	[[ ${lines[0]} = "exec /run.sh: no such file or directory" ]]
 }
+
+# https://github.com/opencontainers/runc/issues/4688
+@test "runc run check default home" {
+	requires root
+	echo 'tempuser:x:14821:14821:tempuser:/home/tempuser:/bin/sh' >>rootfs/etc/passwd
+
+	chown 14821 rootfs/root
+	chmod 700 rootfs/root
```

Comment:
Why are you changing this? Can't we just us `/` as `cwd`, or something that doesn't need to adjust permissions? Am I missing something?

Reply:
I copied them from another test cases, so I removed it now.

Reply:
Thanks!